### DISCUSSION TITLE
Old versions?

### DISCUSSION THREAD

**Discussion by BrenBarn** [hawkeye116477/waterfox-deb-rpm-arch-AppImage#52] [General]
Hi, thanks so much for this useful resource.  It seems the download repository only contains the latest versions of of Classic and G4.  Is there any way to get access to older versions (like G3)?

**Comment by hawkeye116477** [comment]
No.

**Reply by BrenBarn** [reply]
I see.  That is unfortunate, but thanks for your reply.
